src/app/pages/apps/components/app-detail-view/app-detail-view.component.ts

```diff
@@ -16,6 +17,21 @@ import { ApplicationsService } from 'app/pages/apps/services/applications.servic
 import { AvailableAppsStore } from 'app/pages/apps/store/available-apps-store.service';
 import { LayoutService } from 'app/services/layout.service';
 
+const fakeImages = [
+  {
+    srcUrl: 'assets/images/stars-sky-1200w.jpg',
+    previewUrl: 'assets/images/stars-sky-1200w.jpg',
+  },
+  {
+    srcUrl: 'assets/images/stars-sky-800w.jpg',
+    previewUrl: 'assets/images/stars-sky-800w.jpg',
+  },
+  {
+    srcUrl: 'assets/images/stars-sky-400w.jpg',
+    previewUrl: 'assets/images/stars-sky-400w.jpg',
+  },
+];
```

Comment:
FYI, it's unlikely that we'll have separate urls for src and previews. I expect just an array of images.